internal/registry/image.go

```diff
@@ -23,13 +24,28 @@ type ImportOptions struct {
 	RegistryPodPort      string
 }
 
+// for testing
+type utils struct {
+	daemonImage        func(name.Reference, ...daemon.Option) (v1.Image, error)
+	portforwardNewDial func(config *rest.Config, podName, podNamespace string) (httpstream.Connection, error)
+	remoteWrite        func(ref name.Reference, img v1.Image, options ...remote.Option) (rerr error)
```

Comment:
Do we need name of returned value (`rerr`) in this place?

Reply:
nope, it's autogenerated. I will remove it